.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,29 @@
+name: Benchmark
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+  workflow_dispatch:
+
+jobs:
+  benchmark:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v2.1.5
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Run benchmarks
+        run: npm run benchmark
+
+      - name: Delta-T
+        uses: eduardoboucas/delta-t-action@main
```

Comment:
We can transfer this to the `netlify` org if/when we're happy with this approach.